src/main/java/seedu/address/logic/parser/AddNumberOfRoomsParser.java

```diff
@@ -33,4 +32,5 @@ public AddNumberOfRoomsCommand parse(String args) throws ParseException {
         }
         return new AddNumberOfRoomsCommand(numberOfRooms);
     }
```

Comment:
I thought of that, but why would anyone want to input a negative number?

Reply:
by accident? maybe a slip of the finger?

Reply:
okay sure ill add it

Reply:
It is less about whether the user will enter a negative number, and more about how we should secure our application.

Personally, I feel that it is good if we can handle as many exceptional cases (sometimes called edge cases) as possible. Without proper handling, anyone can crash the app simply by entering a negative number. It can be annoying for the user if they accidentally type in a negative number and the app just stops working.

Besides, near the end of the semester, our application will be manually tested for bugs. I am fairly sure the testers will try to use negative numbers as inputs. If they can cause the application to crash, they will flag the problem as a bug and deduct our marks for it. Let's do our best to stop that from happening! 😄